tests/api.v2/routes/experiment.test.js

```diff
@@ -0,0 +1,45 @@
+const express = require('express');
+const request = require('supertest');
+const expressLoader = require('../../../src/loaders/express');
+
+jest.mock('../../../src/api.v2/controllers/experimentController');
+
+describe('tests for experiment route', () => {
+  let app = null;
+
+  beforeEach(async () => {
+    const mockApp = await expressLoader(express());
+    app = mockApp.app;
+  });
+
+  afterEach(() => {
+    /**
+     * Most important since b'coz of caching, the mocked implementations sometimes does not reset
+     */
+    jest.resetModules();
+    jest.restoreAllMocks();
+  });
+
+  it('Creating a new experiment results in a successful response', async (done) => {
```

Comment:
Maybe add tests for invalid responses? E.g. those with empty body, invalid body, invalid method